libraries/rush-lib/src/api/Subspace.ts

```diff
@@ -134,11 +134,6 @@ export class Subspace {
               `The project level configuration file ${project.projectFolder}/.npmrc is no longer valid. Please use a ${subspaceConfigFolder}/.npmrc file instead.`
             );
           }
-          if (FileSystem.exists(`${project.projectFolder}/.pnpmfile.cjs`)) {
-            throw new Error(
-              `The project level configuration file ${project.projectFolder}/.pnpmfile.cjs is no longer valid. Please use a ${subspaceConfigFolder}/.pnpmfile-subspace.cjs file instead.`
```

Comment:
@william2958 Why are we removing this validation?

Seems like we should keep this check, but change the error message to recommend `${subspaceConfigFolder}/.pnpmfile.cjs`

Reply:
Added back